server.js

```diff
@@ -14,12 +14,24 @@ app.use(cors()); // acts as middleware - intermediary between server and request
 
 app.get('/weather', handleGetWeather);
 
+
+
 function handleGetWeather(request, response) {
-  // sends weather array and 200 (ok) response
+  // sends weather array and 200 (ok) .status(200) response
+  console.log(request.query);
```

Comment:
Here, request.query.( ) <--- (this is where you will find the query you made in your front end app)
 
Example: 
- request.query.searchQuery 

- "http://localhost:3001/weather?searchQuery=your_search_parameter"